config.md

```diff
@@ -340,6 +340,52 @@ For Linux-based systems, the `process` object supports the following process-spe
 
     * **`class`** (string, REQUIRED) specifies the I/O scheduling class. Possible values are `IOPRIO_CLASS_RT`, `IOPRIO_CLASS_BE`, and `IOPRIO_CLASS_IDLE`.
     * **`priority`** (int, REQUIRED) specifies the priority level within the class. The value should be an integer ranging from 0 (highest) to 7 (lowest).
+* **`landlock`** (object, OPTIONAL) specifies the Landlock unprivileged access control settings for the container process.
```

Comment:
What happens if the runtime doesn't know this field? I guess in runc it will be completely ignored, right? In that case, I think we at least need to expose landlock support via the features command too.

Reply:
Yes, the runtime will ignore the field. I think it's not necessary to add feature command like other optional features.

Reply:
@Zheaoli why not? This is a security feature, I can see use cases where upper layers want to run enforce this.

Reply:
Sorry for replying late. I misunderstand something before. I think it's necessary to export the feature we support for landlock in features command